### ISSUE TITLE
Disk corruption during installation of Windows 7

### ISSUE THREAD

**Issue by Vort** [bochs-emu/Bochs#384]
I tried to install Windows 7 SP1 x64 to Bochs.
Installation started fine, lots of files were unpacked to virtual disk, but then, after half an hour, error appeared:

> Windows cannot install required files. The file may be corrupt or missing. Make sure all files required for installation are available, and restart the installation. Error code: 0x80070570

![bochs_win7_fail](https://github.com/user-attachments/assets/92bdf9b9-eece-4d87-8162-fc8bfa5fbc00)

I tried to look what happened wrong with disk image by attaching it to VirtualBox, but VirtualBox rejected this file with error "Could not get the storage format of the medium".

Then I attached it to Bochs installation of Windows XP and automatically launched chkdsk immediately started to show errors (tons of them):
![bochs_win7_disk_corruption](https://github.com/user-attachments/assets/19aa4738-194b-48f6-9d7f-e62b695a049c)
![bochs_win7_disk_corruption2](https://github.com/user-attachments/assets/5d1b5416-0c12-4c63-ab30-d4a9102bb8d5)

Can anyone try to reproduce this problem?

I made installation to fresh 30 GB (30720 MB) virtual disk in vmware4 format, which was created with bximage.
Installation stopped when `Windows7.vmdk` file reached size of 4.5 GB.
Here are config and log files from failed installation: [bochs_win7_fail.zip](https://github.com/user-attachments/files/17717725/bochs_win7_fail.zip).

Version: c41a2ae

**Comment by Vort** [comment]
Second installation, with `growing` disk image type instead of `vmware4`, finished successfully.
It means either:
1. Problem is related to disk image type;
2. Problem reproduction is random.

More testing is needed.

**Comment by fuel-pcbox** [comment]
works fine for me with a `flat` disk image :p

**Comment by Vort** [comment]
For third attempt (`vmware4`) I made two changes:
1. I added `Windows7.vmdk` to VirtualBox before I started installation with Bochs.
2. I changed `sync=` from `slowdown` to `none` to make installation faster.

Result is very similar to result from first attempt:
Installation stopped when about 65% of files were extracted. This time with slightly different error message, but at the same mark of 4.5 GB (4 832 821 248 bytes):
![bochs_win7_fail2](https://github.com/user-attachments/assets/43d32446-e1ec-4fa2-b6c3-a14bb3f46f29)

This time VirtualBox provided more information about problematic image file:
>VMDK: inconsistency between grain table and backup grain table

![image](https://github.com/user-attachments/assets/cce7c451-a33c-4ade-95b8-023e9ed36f16)
